napari_animation/animation.py

```diff
@@ -116,6 +116,40 @@ def _set_viewer_state(self, state):
 
         self.viewer.camera.update(camera_state)
         self.viewer.dims.update(state['dims'])
+        self._set_layer_state(state['layers'])
+
+    @property
```

Comment:
Can we do without this, seems overkill to have `self.layers` when it's pretty easy to get `self.viewer.layers`